src/endless/ImageSigningKey.h

```diff
@@ -13,7 +13,6 @@ static const uint8_t endless_public_key_longid[8] = {
 // Endless public key
 static const uint8_t endless_public_key[] = {
     "-----BEGIN PGP PUBLIC KEY BLOCK-----\n"
-    "Version: GnuPG v1\n"
```

Comment:
I checked the parser and it doesn't depend on this header being present (in fact it just skips it).

Reply:
Good to know.